ui/de-lib/src/main/java/org/iplantc/de/commons/client/ErrorHandler.java

```diff
@@ -162,11 +162,11 @@ private static String parseExceptionJson(Throwable caught) {
         }
 
         if (jsonError != null) {
-            String name = JsonUtil.getInstance().getString(jsonError, "name"); //$NON-NLS-1$
-            String message = JsonUtil.getInstance().getString(jsonError, "message"); //$NON-NLS-1$
+            String error_code = JsonUtil.getInstance().getString(jsonError, "error_code"); //$NON-NLS-1$
+            String message = JsonUtil.getInstance().getString(jsonError, "reason"); //$NON-NLS-1$
```

Comment:
You could go the extra mile here and decode an AutoBean instead of JsonObjects, or that could be done later in another PR.